cmd/argocd/commands/account.go

```diff
@@ -143,6 +157,13 @@ func NewAccountGetUserInfoCommand(clientOpts *argocdclient.ClientOptions) *cobra
 	var command = &cobra.Command{
 		Use:   "get-user-info",
 		Short: "Get user info",
+		Example: templates.Examples(`
+			# Get User information
```

Comment:
```suggestion
			# Get User information for the currently logged-in user (see 'argocd login')
```

Maybe a bit more specific?